README.md

```diff
@@ -2,164 +2,14 @@
-
-To track the user's location in the foreground, call:
-
-```js
-cordova.plugins.radar.trackOnce((result) => {
-  // do something with result.status, result.location, result.events, result.user
-});
-```
-
-`err` will be a string, one of:
-
-- `ERROR_PUBLISHABLE_KEY`: the SDK was not initialized
-- `ERROR_PERMISSIONS`: the user has not granted location permissions for the app
-- `ERROR_LOCATION`: location services were unavailable, or the location request timed out
-- `ERROR_NETWORK`: the network was unavailable, or the network connection timed out
-- `ERROR_UNAUTHORIZED`: the publishable API key is invalid
-- `ERROR_SERVER`: an internal server error occurred
-- `ERROR_UNKNOWN`: an unknown error occurred
-
-### Background tracking
-
-Once you have initialized the SDK, you have identified the user, and the user has granted permissions, you can start tracking the user's location in the background.
-
-To start tracking the user's location in the background, call:
-
-```js
-cordova.plugins.radar.startTracking();
-```
-
-Assuming you have configured your project properly, the SDK will wake up while the user is moving (usually every 3-5 minutes), then shut down when the user stops (usually within 5-10 minutes). To save battery, the SDK will not wake up when stopped, and the user must move at least 100 meters from a stop (sometimes more) to wake up the SDK. **Note that location updates may be delayed significantly by iOS [Low Power Mode](https://support.apple.com/en-us/HT205234), by Android [Doze Mode and App Standby](https://developer.android.com/training/monitoring-device-state/doze-standby.html) and [Background Location Limits](https://developer.android.com/about/versions/oreo/background-location-limits.html), or if the device has connectivity issues, low battery, or wi-fi disabled. These constraints apply to all uses of background location services on iOS and Android, not just Radar. See more about [accuracy and reliability](https://radar.io/documentation/sdk#accuracy).**
-
-To stop tracking the user's location in the background (e.g., when the user logs out), call:
-
-```js
-cordova.plugins.radar.stopTracking();
-```
-
-You only need to call these methods once, as these settings will be persisted across app sessions.
-
-To listen for events and errors, you can add event listeners:
-
-```js
-cordova.plugins.radar.onEvents((events, user) => {
-  // do something with events, user
-});
-
-cordova.plugins.radar.onError((err) => {
-  // do something with err
-});
-```
-
-You should remove event listeners when you are done with them:
-
-```js
-cordova.plugins.radar.offEvents();
-
-cordova.plugins.radar.offError();
-```
-
-### Manual tracking
-
-You can manually update the user's location by calling:
-
-```js
-const location = {
-  latitude: 39.2904,
-  longitude: -76.6122,
-  accuracy: 65
-};
-
-cordova.plugins.radar.updateLocation(location, (result) => {
-  // do something with result.status, result.events, result.user
-});
-```
```

Comment:
looks like the README is going to be pretty thin after this PR - I know it's in the docs, but do we want to include some installation / setup instructions here?

Reply:
Just matching https://github.com/radarlabs/react-native-radar. I think we generally want people going to our hosted docs rather than GitHub.